projboard/tests/test_article.py

```diff
@@ -41,7 +41,7 @@ def create_articles(self, create_user, create_subjects):
 
     @pytest.mark.skip
     @pytest.mark.django_db
-    def test_objectCreation(self, create_articles):
+    def test_object_creation(self, create_articles):
```

Comment:
You can remove this function , we check this functionality in migration files.

Reply:
I don't think this change need to be at this PR.
We can create a new issue for not overload on this PR.
WDYT?

Reply:
+1 @Anas-mhameed 
Can be deleted. 
Could be done in different PR also

Reply:
Done